assets/CSS/style.css

```diff
@@ -1,3 +1,667 @@
+html {
+  background: #fff;
+}
+
+body {
+  background-color: #e5e5e5;
+  border-bottom-left-radius: 0;
+  width: 1440px;
+}
+
+.flex-conatiner,
+header {
+  overflow: hidden;
+  background-color: #fff;
+  width: 100%;
+  height: 916px;
+}
+
+header {
+  position: relative;
+}
+
+.logo {
+  overflow: hidden;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-left: 24px;
+  margin-top: 58px;
+  background-color: #fff;
+}
+
+.see-project-btn a {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  text-decoration: none;
+  color: #396df2;
+  width: 120px;
+  height: 40px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border-radius: 8px;
+  background-color: #fff;
+  border: 1px solid #396df2;
+  padding-left: 8px;
+}
```

Comment:
Your buttons are missing the interaction styling. Please add them for all, I'm marking this one just for brevity.

<img width="631" alt="Screen Shot 2022-06-09 at 10 33 06" src="https://user-images.githubusercontent.com/82426682/172886553-1f22d0dd-6a0f-49c3-9b19-27a88e77660c.png">